src/components/menu/menu.tsx

```diff
@@ -53,30 +71,28 @@ export const MobileMenu = () => {
       </div>
       <div className={classNames(styles.mobileMenu, { [styles.open]: open })}>
```

Comment:
I'm not sure about this screen (around 1000px), it doesn't feel right. We probably need to center it and have a full-width button, but it's just a guess.

I suggest we ask Design Team to provide a screen for the mobile menu on SM breakpoint. I see we don't have this in the previous version of the DAO, thus there's no way for us to tell how it should look like.

<img width="864" alt="image" src="https://github.com/user-attachments/assets/e04e78c1-66e8-442a-bff0-0144a0563726">

Reply:
Yeah, there's no design. I will ask about this

Reply:
Actually, after some more thinking I adapted the same approach as on the market - the mobile menu has fixed width on those tablet screens. WDYT?

Reply:
Great call 👌 
Surely that's the only way it makes sense

Reply:
Oh, and since we're doing this we can prevent the page scrolling when menu is open. Like we did on the Market.

```ts
menu.tsx:
<div className={classNames(styles.mobileMenuWrapper, { "mobile-menu-open": open })}>
```
```ts
index.scss:
html:has(.mobile-menu-open) {
  overflow: hidden;
}
```

Reply:
Nice touch. Thanks!